src/Command/BakeCommand.php

```diff
@@ -152,6 +155,19 @@ public function getTemplatePath(Arguments $args, ?string $container = null): str
         return str_replace('/', DIRECTORY_SEPARATOR, $path);
     }
 
+    /**
+     * Creates a new instance of TemplateRenderer with theme set.
+     *
+     * @return \Bake\Utility\TemplateRenderer
+     */
+    public function getTemplateRenderer(): TemplateRenderer
```

Comment:
`get` is not a suitable prefix if the method is going to build a new instance each time. `build`, `create` would be better.

Reply:
Sounds good. The code changed and I left the name.